spec/view_cooks_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+feature "view cooks" do
+  scenario "user can see homepage" do
+    visit root_path
+    expect(page).to have_content "Batch-it!"
+  end
+
+  scenario "user can visit new cook form from homepage" do
+    visit root_path
+    click_button "I am a cook"
```

Comment:
When you click the "I am a cook" button, what do you expect to happen?

Reply:
I've tried to correct this. I'll make sure to check with you I've changed it correctly.